ClosedXML_Tests/Excel/Loading/LoadingTests.cs

```diff
@@ -76,6 +77,16 @@ public void CanSuccessfullyLoadLOFiles()
             }
         }
 
+        [Test]
+        public void InvalidFileThrowsCustomException()
```

Comment:
I think there have to be 3 tests - one for each method throwing an exception - so we won't lose it in case of future refactoring.